static/app/views/explore/logs/utils.tsx

```diff
@@ -503,6 +509,49 @@ export function ourlogToJson(ourlog: TraceItemDetailsResponse | undefined): stri
   return JSON.stringify(copy, null, 2);
 }
 
+export function viewLogsSamplesTarget({
```

Comment:
Can we maybe reuse `getLogsUrl` here?

Reply:
we'll think about how to consolidate more of these ways to link to logs